src/jv.h

```diff
@@ -27,7 +27,10 @@ typedef struct {
   int size;
   union {
     struct jv_refcnt* ptr;
-    double number;
+    struct {
+      double dbl;
+      int64_t int64;
```

Comment:
This is not desirable, unfortunately, as it increases the `sizeof(jv)`.

What I would recommend instead is a new `jv` kind flag that represents a signed 64-bit integer.  Then add `jv_int64()` (creates a `jv` from an `int64_t`), `jv_int64_value()` (returns an `int64_t` given a numeric `jv`).  For backwards compatibility I'd have `jv_get_kind()` return `JV_KIND_NUMBER`, and then add a `jv_get_number_kind()` that reports the numeric kind.  `jv_number_value()` and `jv_int64_value()` should both accept numbers of any kind, and do the appropriate conversion.

This would allow us to also add `uint64_t` as well.

All arithmetic and math functions would continue to operate on `double`s, though perhaps the arithmetic operations could eventually be made to operate on integers (falling back on `double`s in the event of overflow/underflow).